lib/internal/util.js

```diff
@@ -187,3 +189,36 @@ exports.convertToValidSignal = function convertToValidSignal(signal) {
 
   throw new Error('Unknown signal: ' + signal);
 };
+
+const kCustomPromisifiedSymbol = Symbol('util.promisify.custom');
+function promisify(orig) {
+  if (typeof orig !== 'function') {
+    throw new TypeError('The first argument to promisify() must be a function');
+  }
+
+  if (orig && orig[kCustomPromisifiedSymbol]) {
```

Comment:
This truthy test of `orig` is not needed since it is guaranteed to be a function at this point